build.gradle

```diff
@@ -156,7 +156,7 @@ task generatePDF (
         description: 'use pdf as asciidoc backend') {
 
     attributes \
-        'plantUMLDir'         : file('build/docs/images/plantUML/').path
+        'plantUMLDir'         : file("${config.outputPath}/images/plantUML/").path
```

Comment:
I'm not sure if this is correct. The plantUMLDir is only used for PDF-generation. I had to introduce this because otherwise the PDF generator didn't pick the generated images up.
I will test this asap.

Reply:
just wrote a test for it - plantUML still works. Will push my test as soon as it is cleaned up.